src/testing/flow_solver_cases/periodic_cube_flow.cpp

```diff
@@ -167,7 +174,7 @@ void PeriodicCubeFlow<dim, nstate>::compute_unsteady_data_and_write_to_table(
         unsteady_data_table->set_precision(kinetic_energy_string, 16);
         unsteady_data_table->set_scientific(kinetic_energy_string, true);
         // Write to file
-        std::ofstream unsteady_data_table_file(this->unsteady_data_table_filename_with_extension);
+        std::ofstream unsteady_data_table_file(this->all_param.flow_solver_param.unsteady_data_table_filename+".txt");
```

Comment:
should use `unsteady_data_table_filename_with_extension` once you've moved it to this class

Reply:
Fixed in newest commit